### ISSUE TITLE
SVG?

### ISSUE THREAD

**Issue by richschwer** [adrianba/webstandards#3]
SVG is missing and it is currently part of the interactive domain.

**Comment by darobin** [comment]
I don't think that we should consider W3C Domains as a source here. In some ways they are relatively arbitrary. And SVG is not the only thing in Interaction that isn't included (CSS comes to mind :).

My initial instinct would be for the SVG WG to just stay as it is, but I would be interested to hear if anyone has in mind the notion that it would make sense in a web-markup group.

**Comment by adrianba** [comment]
I don't think SVG is missing. My intent isn't to revisit the scope of every working group.

That said, there is a proposal to add canvas 2d to SVG: [w3c/charter-html/issues/14#issuecomment-103618353](https://github.com/w3c/charter-html/issues/14#issuecomment-103618353). Perhaps that would make this more of a "2D graphics" WG?
